web_ir_actions_act_view_reload/static/src/js/web_ir_actions_act_view_reload.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+// Copyright 2018 Modoolar <info@modoolar.com>
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
The module uses LGPL license according to the module manifest. I think OCA always uses AGPL.

Reply:
I didn't change anything here.
@pnajman-modoolar can we consider it a mistake and change it to LGPL to be consistent with the manifest?

Reply:
@guewen sure

Reply:
@guewen , @tarteo guys, What's the proper licence for OCA modules `LGPL-3` or `AGPL-3` ?
I'm asking because [here](https://github.com/OCA/web/blob/13.0/web_dialog_size/__manifest__.py) we have AGPL-3 and [here](https://github.com/OCA/web/blob/13.0/web_responsive/__manifest__.py) LGPL-3.

Reply:
It's the one you want to choose to fit your needs. Boths are valid in OCA.

Reply:
Yep. I updated the file to match the manifest.